lib/routes/hrbeu/sec/list.ts

```diff
@@ -0,0 +1,84 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+const rootUrl = 'http://sec.hrbeu.edu.cn';
+
+export const route: Route = {
+    path: '/sec/:id',
+    categories: ['university'],
+    example: '/hrbeu/sec/xshd',
+    parameters: { id: '栏目编号，由 `URL` 中获取。' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['sec.hrbeu.edu.cn/:id/list.htm'],
+        },
+    ],
+    name: '船舶工程学院',
+    maintainers: ['Chi-hong22'],
+    handler,
+    description: `  | 学院要闻 | 学术活动 | 通知公告 | 学科方向 |
+                    | :------: | :------: |:------: | :------: |
+                    |   xyyw   |   xshd   |  229   |   xkfx   |`,
```

Comment:
```suggestion
    description: `| 学院要闻 | 学术活动 | 通知公告 | 学科方向 |
| :------: | :------: |:------: | :------: |
|   xyyw   |   xshd   |  229   |   xkfx   |`,
```
Indent the description with either 0 or 2 spaces, otherwise the markdown content won't be rendered properly